### ISSUE TITLE
Uncaught TypeError: Cannot read property 'nr' of undefined

### ISSUE THREAD

**Issue by codehz** [vue-comps/vue-clusterize#9]
clusterize.js:374 

``` JavaScript
        absI = absIs[n];
        relI = absI % 3;
        if (this.clusters[relI].nr !== absI || repaint) {
```

and i found that rell is -1

clusterVisible == 1 and clustersCount == 2 and clustersBelow == 0
absIs = [-1, 0, 1]
absI = -1
relI = absI % 3; //=-1

I think this case you can just detect it and return because all item are rendered, nothing need to do..

**Comment by paulpflug** [comment]
should be solved in 0.5.0 by always enforce clustersCount > 2

I changed the usage in 0.5.0. Hopefully easier :smile:

**Comment by codehz** [comment]
fixed
